### PR TITLE
Fixing Documents and add methods to list

### PR DESCRIPTION
**Pull request by Lee-W**
* Add compile guide to "Contributing to Batavia" page
* Fix error in "Contributing to the documentation" page
* Remove the unused VirtualMachine.js lines
* Remove copy from Tuple.js
* Add count to List.js

### CODE REVIEW

File: tests/datatypes/test_list.py [hunk 1]
```diff
@@ -372,6 +371,64 @@ def test_clear_empty_list(self):
             print(l)
         """)
 
+    def test_count(self):
+        self.assertCodeExecution("""
+        x = (1, 2, 2, 3)
```
Comment (freakboy3742) [inline tests/datatypes/test_list.py]: Is this meant to be a list, rather than a tuple?

File: tests/datatypes/test_list.py [hunk 2]
```diff
@@ -372,6 +371,64 @@ def test_clear_empty_list(self):
...
+        except TypeError as e:
+            print(e)
+        try:
+            x.count()
+        except TypeError as e:
+            print(e)
+        """)
+
+    def test_index(self):
+        self.assertCodeExecution("""
```
Comment (freakboy3742) [inline tests/datatypes/test_list.py]: Is this meant to be a list, rather than a tuple?

**Comment by chairco** [comment]
@Lee-W Do you mind I cherry-pick your commit `671d144`,  `4c1b3ca`,  `fcbb93d` at #568 PR? 
 we edit the same method at List.js and test_list.js. if can merge at one branch I think @freakboy3742 can more easily to review and merge into master? @freakboy3742 Is a good way? or do you have any suggestion for us?

**Comment by Lee-W** [comment]
@chairco Sounds great! Do I have to make any modification?
BTW, I've found where this test failed. It's due to the deleted copy function in Tuple. I'm planning to fix it this weekend.

**Comment by chairco** [comment]
Yes, I think after merge your commit success can git revert to `2bb7f55` and go ahead with the implement in the Tuple.js  :honeybee: :honeybee: :dancers: thank you.

**Pull request merged**